### ISSUE TITLE
Remove decimal place from speed limits palette?

### ISSUE THREAD

**Issue by aubergine10** [krzychu124/Cities-Skylines-Traffic-Manager-President-Edition#651]
There doesn't seem to be much benefit having the decimal place in the km/h (or MPH if in km/h mode) captions under the speed icons:

![mph](https://camo.githubusercontent.com/a647080f1c74639fd51eb066b4be7ae9eca3c046/68747470733a2f2f692e696d6775722e636f6d2f426345666236642e706e67)

IMO, visually at least, we should just round up the numbers to nearest whole number and ditch the `.x` part for cleaner looking display.

**Comment by kvakvs** [comment]
I am 99% sure it never was there during the original speed limits PR review and merge.
Something was added later which affected it.

**Comment by aubergine10** [comment]
Yup, seems to be regression - here's image from the PR where MPH speeds were first added:

![image](https://user-images.githubusercontent.com/1386719/73810317-0d49d900-47ce-11ea-8ae3-d60a07058812.png)
